roslibrust/examples/basic_publisher.rs

```diff
@@ -1,19 +1,16 @@
-use log::*;
-use roslibrust::ClientHandle;
-
+#[cfg(feature = "rosbridge")]
 roslibrust_codegen_macro::find_and_generate_ros_messages!("assets/ros1_common_interfaces/std_msgs");
 
 /// This example creates a client, and publishes a message to the topic "talker"
 /// Running this example at the same time as subscribe_and_log will have the two examples
 /// pass messages between each other.
 /// To run this example a rosbridge websocket server should be running at the deafult port (9090).
+#[cfg(feature = "rosbridge")]
 #[tokio::main(flavor = "multi_thread")]
-async fn main() -> Result<(), anyhow::Error> {
-    simple_logger::SimpleLogger::new()
-        .with_level(log::LevelFilter::Debug)
-        .without_timestamps() // required for running wsl2
-        .init()
-        .unwrap();
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
Why the change from `anyhow`- just to get rid of the dependency?

Reply:
Yeah mainly just trying to simplify examples and use the least amount of extra things in them.